### ISSUE TITLE
1.12 amendment to plant growth level display

### ISSUE THREAD

**Issue by Zygus42** [McJty/TheOneProbe#246]
theoneprobe-1.12-1.4.22
forge-1.12.2-14.23.3.2685-installer-win

When you look at a normal plant such as wheat, you see the growth stages, but that info line goes away when the plant is fully grown.  However, when you also have mods on where one probe is not displaying the growth level, that info line is also absent, leading you to harvest them even though they are not fully grown.  

I see possible 2 solutions to this:
1) make the info on normal plants say something like "mature, ready to harvest" in green, or
2) when the plant is unknown to one probe, display something like: "unknown growth level" in red

**Comment by McJty** [comment]
Sorry I'm confused. I don't understand what you mean by this or what the problem is exactly?

**Comment by DrManganese** [comment]
The probe shows info about any crop that extends the default vanilla crop class. So if a modded crop isn't showing growth percentage there is no way for the probe to know if it even is a crop.

**Comment by Zygus42** [comment]
I figured that was probably the case, so I see option 1 above as the most plausible option that the most people will understand.  

Obviously ideally all crops would count as normal crops and be seen as such, but we just started a new world, and people planted all the stuff they found next to each other and they discovered that this was a problem for them, as they're new to One Probe (used waila and hwaila and WIT) in the past.  So they were harvesting crops when One Probe had no more growth information line, only to find that the crops were not ready to be harvested yet.
